cypress/README.md

```diff
@@ -0,0 +1,8 @@
+# Cypress Setup Path
+
+- [x] install cypress
+- [x] set up an auto-signing web3 provider that works well with the test chain
+- [x] create a static seed that covers all use cases and patterns
+- [x] set up a more advanced test script to showcase cypress' functionality
+- [x] configure a docker-compose file that doesn't clobber the dev container setup
```

Comment:
This seems like a todo list of stuff you completed vs a readme on how cypress tests should be run